src/passes/hash-stringify-walker.cpp

```diff
@@ -204,6 +204,9 @@ std::vector<SuffixTree::RepeatedSubstring> StringifyProcessor::filter(
     void walk(Expression* curr) {
       hasFilterValue = false;
       Super::walk(curr);
+      while (!controlFlowQueue.empty()) {
+        dequeueControlFlow();
+      }
```

Comment:
You'll probably laugh, but can we pull this out into a single function like `flushControlFlowQueue`? That way we can avoid duplicating the loop in multiple places.

Reply:
done